### ISSUE TITLE
Remove all mentions of cxbe 

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/Cxbx-Reloaded#134]
Cxbe doesn't exist anymore, but is still mentioned in all copyright comment headers.

Remove all these and possible other occurrences of Cxbe from our code base.

**Comment by x1nixmzeng** [comment]
This is related to #89. There is some old documentation to cleanup too.

I'm looking to find some free time to do it myself!

**Comment by PatrickvL** [comment]
Wait a moment, if I'm not mistaken someone on a fork already started working on this

**Comment by PatrickvL** [comment]
https://github.com/DisorderIy/Cxbx-Reloaded/commit/d331951ecb1600a2aa75922738d0fdec0e885651 it was, although that got deleted afterwards... ?

**Comment by x1nixmzeng** [comment]
We need to make a tutorial on how to branch. Several forks aren't using Git correctly - rather, in a way that we can pilfer certain commits! It's also making our history a bit hard to follow

**Comment by PatrickvL** [comment]
Oh please, do write up or find an applicable tutorial or guideline document for us to use, we're in dire need of one!

**Comment by PatrickvL** [comment]
Does this help? https://help.github.com/articles/setting-guidelines-for-repository-contributors/

**Comment by PatrickvL** [comment]
The Cxbe removal returned here : https://github.com/DisorderIy/Cxbx-Reloaded/commit/0d094e71b7dbb2f5a15928661944492b3c06efa7

**Comment by BenNottelling** [comment]
I took care of this on my fork: https://github.com/BenNottelling/Cxbx-Reloaded
Putting a pull request now
